lib/server-dev.js

```diff
@@ -10,15 +10,14 @@ var config  = require('./config');
  */
 var server = listen(3000);
 emitter({app: server});
-open('http://publish.prx.dev');
```

Comment:
I've started to find this annoying as well.  Stop opening that page over and over!